src/Microsoft.DotNet.Build.Tasks/PrereleaseResolveNuGetPackageAssets.cs

```diff
@@ -47,6 +48,8 @@ public sealed class PrereleaseResolveNuGetPackageAssets : Task
         private readonly List<ITaskItem> _contentItems = new List<ITaskItem>();
         private readonly List<ITaskItem> _fileWrites = new List<ITaskItem>();
 
+        private readonly List<string> _packageFolders = new List<string>();
```

Comment:
In the commit description for 317aa8a6d14fa49dd7b60bb5db2e0d0e052d3904 can you note that it's a port of https://github.com/NuGet/NuGet.BuildTasks/commit/d37c9176cf0b93ab86367675719fa61bbf757d2d?